sdk/core/src/request.rs

```diff
@@ -13,6 +13,19 @@ pub enum Body {
     SeekableStream(Box<dyn SeekableStream>),
 }
 
+impl Body {
+    pub fn len(&self) -> usize {
```

Comment:
While these functions are pretty self-explanatory, I think it's a good habit to always add documentation for all public methods.